### PR TITLE
Fix relation uses in knowledge (#7479)

### PR DESCRIPTION
**Pull request by lndrtrbn**
### Proposed changes

* Add uses relation where it was missing (based on STIX documentation)

### Related issues

* #7479 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/observations/infrastructures/InfrastructureKnowledge.tsx [hunk 1]
```diff
@@ -243,7 +248,7 @@ const InfrastructureKnowledge = ({ infrastructure }: { infrastructure: Infrastru
           element={
             <EntityStixCoreRelationships
               entityId={infrastructureData.id}
-              stixCoreObjectTypes={['Tool']}
+              stixCoreObjectTypes={['Tool', 'uses']}
               relationshipTypes={['hosts']}
```
Comment (JeremyCloarec) [inline opencti-platform/opencti-front/src/private/components/observations/infrastructures/InfrastructureKnowledge.tsx]: uses should be put in the relationshipTypes

Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/private/components/observations/infrastructures/InfrastructureKnowledge.tsx]:
```suggestion
              stixCoreObjectTypes={['Tool']}
              relationshipTypes={['hosts', 'uses']}
```

**Comment by JeremyCloarec** [comment]
Switching to allDirections for malwares in infrastructure fixes the display issue. But it creates an inconsistency with the relation creation behavior: when creating relations, we can only create relation from the infrastructure to the malware (so the controls, delivers and hosts relations).
What's the expected functional behavior on theses views?

**Comment by SamuelHassine** [comment]
> Switching to allDirections for malwares in infrastructure fixes the display issue. But it creates an inconsistency with the relation creation behavior: when creating relations, we can only create relation from the infrastructure to the malware (so the controls, delivers and hosts relations). What's the expected functional behavior on theses views?

Yes it is, it will be reworked later.

**Pull request merged**